rep-2009.rst

```diff
@@ -422,29 +399,26 @@ In addition, if it turns out to be necessary, the simpler approach can always be
 To Take a Centralized or Decentralized Approach
 -----------------------------------------------
 
-The specification above takes a decentralized approach to negotiating publishers and subscribers.
-That is, each negotiating publisher and subscriber negotiates its own selected message type.
-It is also possible to take a centralized approach, where all publishers and subscribers broadcast their preferences to a higher system that decides the selected message types.
+The specification above takes a decentralized approach to negotiating publishers and subscriptions.
+That is, each negotiating publisher and subscription negotiates its own selected message type.
+It is also possible to take a centralized approach, where all publishers and subscriptions broadcast their preferences to a higher system that decides the selected message types.
 
 The primary advantage of a decentralized approach is that it is easier to implement, especially given the greedy approach that we are using in computing the selected message type.
 If we wanted to find the optimal selected message types taking the entire system into account, we would most likely have to implement a centralized approach, which would have a full understanding of the entire system before making a decision.
 
 
-Extending Lifecycle Nodes
--------------------------
+Using in conjunction with Lifecycle Nodes
+-----------------------------------------
 
 Very much of the stateful behavior that is required for the negotiation process is implemented in lifecycle nodes.
-It also allows us to turn off nodes and free up the nodes resources should another task require those resources.
-Thus it was an easy decision to extend the lifecycle nodes to work with the negotiation process.
-However, the type negotiation feature described in this REP will be in a separate package from the lifecycle nodes to avoid changing the lifecycle nodes.
+Thus, while this REP does not prescribe using lifecycle nodes for negotiation, it makes sense for users to combine these two features when implementing nodes.
 
 
-Putting this Implementation in ``rclcpp`` or ``rcl``
-----------------------------------------------------
+Putting this implementation in the ROS 2 core or in a separate project
```

Comment:
It seems like we're using inconsistent casing for titles.

Reply:
Good call.  I used upper-case as needed, and also shortened it a bit.